### ISSUE TITLE
When using local tree database, jageocoder always require to set environment variable `JAGEOCODER_DB2_DIR`.

### ISSUE THREAD

**Issue by miettal** [t-sagara/jageocoder#29]
https://github.com/t-sagara/jageocoder/blob/main/jageocoder/module.py#L70

This is breaking change and decreases API calling portability.
I feel it better to has fallback function. kind ↓

```
diff --git a/jageocoder/module.py b/jageocoder/module.py
index 4d114d5..15b6f11 100644
--- a/jageocoder/module.py
+++ b/jageocoder/module.py
@@ -61,25 +61,22 @@ def init(db_dir: Optional[os.PathLike] = None,
     if _tree:
         del _tree

-    _url = None
-    _db_dir = None
     if db_dir is not None:
         _db_dir = db_dir
-    elif url is not None and mode == 'r':
-        _url = url
-    elif os.environ.get('JAGEOCODER_DB2_DIR'):
+    else:
         _db_dir = get_db_dir(mode=mode)

-    if _db_dir is None and _url is None:
+    if url is not None and mode == 'r':
+        _url = url
+    else:
         _url = os.environ.get('JAGEOCODER_SERVER_URL')

-    if _db_dir:
+    if db_dir is not None and url is not None:
         _tree = AddressTree(db_dir=_db_dir, mode=mode, debug=debug)
     elif _url:
         _tree = RemoteTree(url=_url, debug=debug)
     else:
-        raise JageocoderError(
-            "Neither 'db_dir' nor 'url' could be determined.")
+        _tree = AddressTree(db_dir=_db_dir, mode=mode, debug=debug)

     set_search_config(**kwargs)
```

**Comment by miettal** [comment]
I fogot to post jagecoder version of this behavior.
this behavior is available from jagecoder==2.1.6

**Comment by t-sagara** [comment]
This bug has been fixed in commit https://github.com/t-sagara/jageocoder/commit/79ae508d33842877656b11c0e0d997a196289000

If the environment variable JAGEOCODER_DB2_DIR is not specified, it refers to the database installed in the local Python environment.

**Comment by t-sagara** [comment]
Fixed in v2.1.7.
